spec/facility_spec.rb

```diff
@@ -1,26 +1,156 @@
+      @facility_2.add_service('New Drivers License')
+      @facility_2.add_service('Renew Drivers License')
+      @facility_2.add_service('Vehicle Registration')
+      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License'])
+      expect(@facility_1.services).to eq(['New Drivers License', 'Vehicle Registration'])
+      expect(@facility_2.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end   
+  end
+
+  describe '#register_vehicle' do
+    it 'can register vehicles' do
+      expect(@facility_1.register_vehicle(@cruz)).to eq(nil)
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@cruz.plate_type).to eq(nil)
+      expect(@cruz.registration_date).to eq(nil)
+      expect(@facility_1.collected_fees).to eq(0)
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(100)
+      expect(@facility_1.registered_vehicles).to eq([@cruz]) 
+
+      expect(@camaro.plate_type).to eq(nil)
+      expect(@camaro.registration_date).to eq(nil)
+      expect(@facility_1.register_vehicle(@camaro)).to eq([@cruz, @camaro])
+      expect(@camaro.plate_type).to eq(:antique)
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(125)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro]) 
+
+      expect(@bolt.plate_type).to eq(nil)
+      expect(@bolt.registration_date).to eq(nil)
+      expect(@facility_1.register_vehicle(@bolt)).to eq([@cruz, @camaro, @bolt])
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@bolt.registration_date).to eq(Date.today)
+      expect(@facility_1.collected_fees).to eq(325)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt]) 
+    end
+  end   
+
+  describe '#administer_written_test' do
+    it 'administer a written test' do
+      expect(@facility_1.administer_written_test(@registrant_1)).to eq(false)
+      @facility_1.add_service('Written Test')
+      expect(@facility_1.administer_written_test(@registrant_1)).to eq(true)
+      expect(@registrant_1.license_data[:written]).to eq(true)
+
+      expect(@facility_1.administer_written_test(@registrant_2)).to eq(false)
+      @registrant_2.earn_permit
+      expect(@facility_1.administer_written_test(@registrant_2)).to eq(true)
+      expect(@registrant_2.license_data[:written]).to eq(true)
+
+      expect(@facility_1.administer_written_test(@registrant_3)).to eq(false)
+      @registrant_3.earn_permit
+      expect(@facility_1.administer_written_test(@registrant_3)).to eq(false)
+      expect(@registrant_3.license_data[:written]).to eq(false)
+      end
+    end   
+  
+  describe '#administer_road_test' do
+    it 'administer a road test' do
```

Comment:
Here too! Multiple it statements in the same describe block would be great to make this more readable